### ISSUE TITLE
2.2.4 Beta - Nether rendered entities exhausts system

### ISSUE THREAD

**Issue by og1996** [Beyond-Reality/BeyondRealityModPack#196]
In the 2.2.4 beta, I'm getting horrible lag in the nether. When initially entering the nether, things are fine (also, saving/reloading similar behavior). Within about 10 seconds, the total entities counter starts going up several hundred entities per second without stopping. You can see from the screenshots that the third line E counter skyrockets. Conversely, FPS drops as entities increase. Entities increase until MC runs out of memory. There is no crash report provided, nor error messages on the console. The only changes to the pack are PrimitiveMobs is disabled and I have AromaBackup running. Oddly enough, in the nether, AromaBackup seems to run about every 30 seconds, versus 30 minutes in the overworld. This is all since 2.2.4. This was a previously generated 2.2.3C world.

![2015-02-01_15 50 34](https://cloud.githubusercontent.com/assets/5308959/5994299/96dbe204-aa2a-11e4-9053-55ceb1b0bf34.png)
![2015-02-01_15 51 30](https://cloud.githubusercontent.com/assets/5308959/5994298/96db065e-aa2a-11e4-857f-9766835885f0.png)
![2015-02-01_15 51 41](https://cloud.githubusercontent.com/assets/5308959/5994297/96d8f9d6-aa2a-11e4-86a6-7401999ae15a.png)

**Comment by Dominance** [comment]
Can you screen shot the COFH kill all command. Would love to see what its removing. Do Not think and updated mods this version adds any new entities.

**Comment by og1996** [comment]
![2015-02-03_18 40 25](https://cloud.githubusercontent.com/assets/5308959/6033555/6a04155c-abd4-11e4-9af2-baa14957b98b.png)
Spamming /cofh killall every second just kills between 50 and 150 entities. There's a doom like dungeon underneath me and the spawners are spewing forth new entities every tick. Seems to be no rhyme or reason as to which monsters are killed the most with each killall.

**Comment by Dominance** [comment]
I would say aroma backup is exhausting your resources. As for the massive amounts of mobs spawning nothing in that department has changed.

**Comment by og1996** [comment]
I've been using AromaBackup successfully for quite sometime with your pack -- it has a tendency to crash and corrupt save files. Since 2.2.4, I've noticed AromaBackup is firing off every 5 minutes or so, even though its set to 30 minutes. I'll try removing it and see if that clears it up the spawn problem. Only happens in the nether, which is odd.

**Comment by og1996** [comment]
AromaBackup is the issue. Removing it causes normal amounts of entities. Like I said, it had been working fine prior to 2.2.4, so now I need to find a replacement automated backup solution. :)

**Comment by JHallman** [comment]
I'm having this same issue.  I don't have, nor have had, any Aroma mods installed.  It seems to happen in certain parts of the Nether; on some worlds I can fly for a while normally until the entitie count skyrockets.  Other world the count goes up as soon as a load in to the nether.

I tried removing BoP as I suspected it might have been a certain biome, but that didn't change anything.

The entity count will continue to climb, it has reached over 20k. cofh killall has no effect either.

I have looked through logs and Opis, but I havent been able to locate any spams or even numbers resembling the entity count.  

I can provide any more information that might be needed, I'm just at a loss as to what to look for.
